UE/Application/Ports/UserPort.cpp

```diff
@@ -67,4 +67,17 @@ void UserPort::showTalking()
     gui.setCallMode();
 }
 
+void UserPort::showPeerUserDisconnected()
+{
+    logger.logInfo("UserPort::showPeerUserDisconnected");
+    this->showAlert("Peer User was disconnected from BTS");
+}
+
+void UserPort::showAlert(std::string message)
+{
+    logger.logInfo("UserPort::showAlert: ", message);
+    IUeGui::ITextMode &alertMode = gui.setAlertMode();
+    alertMode.setText(message);
+}
```

Comment:
I wrote a similar method in: 
https://github.com/Koci0/Nokia-PK-2021/blob/7ca47a54718a6bc46f4fd9688b60595787116767/UE/Application/Ports/UserPort.cpp#L61-L69
Maybe we can use one of them. In my method user must click red button to show main menu. What do you think?

Reply:
Okay, I will create an empty method with the same signature, so we will avoid some merging issues later on